### PR TITLE
docs: rewrite Windows install and bootstrap guide

### PR DESCRIPTION
**Pull request by laszlocsomor**
Simplify the installation and bootstrapping
instruction for Windows.

Fixes https://github.com/bazelbuild/bazel/issues/4784

Change-Id: I564cd6136ac5b2f0661665765b9bb488e23fe6b0

### CODE REVIEW

File: site/docs/install-windows.md [hunk 1]
```diff
@@ -5,69 +5,40 @@ title: Installing Bazel on Windows
...
+that's on your `%PATH%` or add its directory to your `%PATH%`. This way you can run Bazel by
+typing `bazel` in any directory, without typing out the full path.
 
-Go to Bazel's [GitHub releases page](https://github.com/bazelbuild/bazel/releases)
-and download the Windows binary<sup>1</sup>: `bazel-<version>-installer-windows-x86_64.sh`.
-
-For convenience, move the binary to a directory that's on your `%PATH%`. This
-way you can run Bazel by typing `bazel` in any directory, without typing out the
-full path. That said, you may put the binary anywhere on your filesystem.
+**Note:** Bazel includes an embedded JDK so you don't need to install one separately. If you want
```
Comment (meteorcloudy) [inline site/docs/install-windows.md]: FYI, we'll no longer provide "without embedded JDK" binaries.
See https://github.com/bazelbuild/continuous-integration/issues/269

File: site/docs/install-compile-source.md [hunk 2]
```diff
@@ -5,57 +5,102 @@ title: Compiling Bazel from Source
 
 # <a name="compiling-from-source"></a>Compiling Bazel from Source (bootstrapping)
 
-You can build Bazel from source without using an existing Bazel binary by
-doing the following:
+You can build Bazel from source without using an existing Bazel binary.
 
-### 1.  Ensure that JDK 8, Python, Bash, zip, and the usual C++ build toolchain are installed on your system.
+### 1.  Install prerequisites
```
Comment (meteorcloudy) [inline site/docs/install-compile-source.md]: We don't have this section for macOS, is that OK?

Comment (laszlocsomor) [reply site/docs/install-compile-source.md]: Good point. I don't know the instructions for macOS, so I'm not modifying that section substantially.

File: site/docs/install-windows.md (hunk 1 above)
Comment (laszlocsomor) [reply site/docs/install-windows.md]: Good to know, thanks! I'll remove this section once that PR is merged.

**Pull request closed without merging**